lib/roo_on_rails/tasks/db.rake

```diff
@@ -23,5 +23,6 @@ end
   db:migrate:down
   db:rollback
 ].each do |task|
+  break unless defined?(ActiveRecord)
```

Comment:
I feel like we can ignore this entire file unless we have ActiveRecord right? So wrapping it in a `if defined?(ActiveRecord)` would almost feel better? Or at least those task enhancing functionality would be best wrapped in this `if` block.